Kubernetes/legos/k8s_check_service_pvc_utilization/README.md

```diff
@@ -6,15 +6,15 @@
 This check fetches the PVC associated with a given service, determines its utilized size, and then compares it to its total capacity. If the used percentage exceeds the provided threshold, it triggers an alert.
 
 ## Lego Details
-	k8s_check_service_pvc_utilization(handle, service_name: str = "", namespace: str = "", threshold: int = 80)
+	k8s_check_service_pvc_utilization(handle, services_to_check_pvc_utilzation: list, namespace: str = "", threshold: int = 80)
```

Comment:
Rename `services_to_check_pvc_utilzation` to `core_services` for this and other checks in this PR